### PR TITLE
Cloud cost changes + tests

### PR DESCRIPTION
**Pull request by Pramaganus**
Changes to SDK regarding cloud costs + tests added.

### CODE REVIEW

**Comment by Pramaganus** [comment]
@tylerwoonton

File: src/Billing/Entities/CloudCost.php [hunk 1]
```diff
@@ -7,15 +7,22 @@
 /**
  * @property int $id
  * @property int $serverId
- * @property \DateTime $startAt
- * @property \DateTime $endAt
+ * @property \DateTime $billing_start
```
Comment (tylerwoonton) [inline src/Billing/Entities/CloudCost.php]:
```suggestion
 * @property \DateTime $billingStart
```

File: src/Billing/Entities/CloudCost.php [hunk 2]
```diff
@@ -7,15 +7,22 @@
...
+ * @property \DateTime $billing_end
```
Comment (tylerwoonton) [inline src/Billing/Entities/CloudCost.php]:
```suggestion
 * @property \DateTime $billingEnd
```

File: src/Billing/Entities/Resource.php [hunk 3]
```diff
@@ -6,9 +6,19 @@
 
 /**
  * @property string $type
- * @property int $value
+ * @property int $quantity
+ * @property int $price
+ * @property string $period
+ * @property float $usage_since_last_invoice
```
Comment (tylerwoonton) [inline src/Billing/Entities/Resource.php]:
```suggestion
 * @property float $usageSinceLastInvoice
```

File: src/Billing/Entities/Resource.php [hunk 4]
```diff
@@ -6,9 +6,19 @@
...
+ * @property float $cost_since_last_invoice
```
Comment (tylerwoonton) [inline src/Billing/Entities/Resource.php]:
```suggestion
 * @property float $costSinceLastInvoice
```

File: src/Billing/Entities/Resource.php [hunk 5]
```diff
@@ -6,9 +6,19 @@
...
+ * @property float $usage_for_period_estimate
```
Comment (tylerwoonton) [inline src/Billing/Entities/Resource.php]:
```suggestion
 * @property float $usageForPeriodEstimate
```

File: src/Billing/Entities/Resource.php (hunk 3 above)
Comment (tylerwoonton) [reply src/Billing/Entities/Resource.php]: Also, Is this a float or an int?

File: src/Billing/Entities/Resource.php (hunk 5 above)
Comment (tylerwoonton) [reply src/Billing/Entities/Resource.php]: Same question about if this is a float or an int?

File: src/Billing/Entities/Resource.php [hunk 6]
```diff
@@ -6,9 +6,19 @@
...
+ * @property float $cost_for_period_estimate
```
Comment (tylerwoonton) [inline src/Billing/Entities/Resource.php]:
```suggestion
 * @property float $costForPeriodEstimate
```

File: src/Billing/Entities/Resource.php [hunk 7]
```diff
@@ -6,9 +6,19 @@
...
+ * @property \DateTime $billing_start
```
Comment (tylerwoonton) [inline src/Billing/Entities/Resource.php]:
```suggestion
 * @property \DateTime $billingStart
```

File: src/Billing/Entities/Resource.php [hunk 8]
```diff
@@ -6,9 +6,19 @@
...
+ * @property \DateTime $billing_end
```
Comment (tylerwoonton) [inline src/Billing/Entities/Resource.php]:
```suggestion
 * @property \DateTime $billingEnd
```

File: src/Billing/Entities/Resource.php [hunk 9]
```diff
@@ -6,9 +6,19 @@
...
+ * @property \DateTime $billing_due_date
```
Comment (tylerwoonton) [inline src/Billing/Entities/Resource.php]:
```suggestion
 * @property \DateTime $billingDueDate
```

File: src/Billing/Entities/Resource.php [hunk 10]
```diff
@@ -6,9 +6,19 @@
...
+ * @property boolean $billing_item_bill_complete
```
Comment (tylerwoonton) [inline src/Billing/Entities/Resource.php]: This property doesn't exist anymore.

**Comment by tylerwoonton** [comment]
@Pramaganus I've raised some suggestions about the annotations but I want to check if the properties need to be in the `CloudCost` entity as well as the `Resource` entity.

**Pull request merged**